test/funnels/automation/StableSwapper.t.sol

```diff
@@ -82,38 +84,139 @@ contract StableSwapperTest is DssTest, TestUtils {
         checkAuth(address(stableSwapper), "StableSwapper");
     }
 
+    function testKissDiss() public {
+        address testAddress = address(0x123);
+
+        assertEq(stableSwapper.buds(testAddress), 0);
+        vm.expectEmit(true, true, true, true);
+        emit Kissed(testAddress);
+        stableSwapper.kiss(testAddress);
+        assertEq(stableSwapper.buds(testAddress), 1);
+        vm.expectEmit(true, true, true, true);
+        emit Dissed(testAddress);
+        stableSwapper.diss(testAddress);
+        assertEq(stableSwapper.buds(testAddress), 0);
+
+        stableSwapper.deny(address(this));
+
+        vm.expectRevert("StableSwapper/not-authorized");
+        stableSwapper.kiss(testAddress);
+        vm.expectRevert("StableSwapper/not-authorized");
+        stableSwapper.diss(testAddress);
+    }
+
+    function testPermitForbid() public {
+        address testAddress = address(0x123);
+        stableSwapper.kiss(address(this));
+
+        assertEq(stableSwapper.bots(testAddress), 0);
+        vm.expectEmit(true, true, true, true);
+        emit Permit(testAddress);
+        stableSwapper.permit(testAddress);
+        assertEq(stableSwapper.bots(testAddress), 1);
+        vm.expectEmit(true, true, true, true);
+        emit Forbid(testAddress);
+        stableSwapper.forbid(testAddress);
+        assertEq(stableSwapper.bots(testAddress), 0);
+
+        stableSwapper.diss(address(this));
+
+        vm.expectRevert("StableSwapper/non-facilitator");
+        stableSwapper.permit(testAddress);
+        vm.expectRevert("StableSwapper/non-facilitator");
+        stableSwapper.forbid(testAddress);
```

Comment:
The new standard would say this should be checked in testModifiers.

Reply:
Are you sure? can you give an example?

Reply:
Well we have been using `testModifiers`, not sure if calling it a new standard is accurate though. To be honest not strong feeling here.